account_credit_control/models/credit_control_line.py

```diff
@@ -232,10 +234,11 @@ def create_or_update_from_mv_lines(self, lines, level, controlling_date,
                 ('move_line_id', '=', move_line.id),
                 ('policy_level_id', '=', level.id),
                 ('state', '=', 'draft'),
-                ('id', '!=', line.id),
             ])
-            if previous_drafts:
-                previous_drafts.write({'state': 'ignored'})
+            lines_to_write = lines_to_write | previous_drafts
+
+        new_lines = self.create(lines_to_create)
```

Comment:
This line in `create` also affects a lot the performance:

https://github.com/OCA/credit-control/blob/01697bf3f5520e5c499b8cd769202cc6248c2aae/account_credit_control/models/credit_control_line.py#L268

Maybe a related field?

Reply:
Not quite sure it can be related, but what about https://github.com/OCA/credit-control/pull/33/commits/d9a95c5f64bed1f05d6ea1ece9bdf63fa52e2816 ?